docs/endpoints/multi_user.rst

```diff
@@ -330,6 +326,17 @@ A heartbeat occurs every 30s; if ``idle_heartbeats_hard`` is set to 7, and no ta
 or results move (i.e., tasks received from the web service or results received from
 workers), then the endpoint will shutdown after 3m30s (7 × 30s).
 
+Every template also has access to the following variables:
+
+- ``user_runtime``: Contains information about the runtime that the user used when
+  submitting the task request, such as Python version. See |UserRuntime|_ for a complete
+  list of available information.
+
+- ``parent_config``: Contains the configuration values of the parent MEP. Can be helpful
+  in situations involving Python-based configuration files.
+
+These are reserved words and their values cannot be overidden by the user or admin.
+
```

Comment:
Suggest adding an a description or example of what happens if one tries to override these.